src/scss/04-patterns/06-utilities/_provider-login-button.scss

```diff
@@ -0,0 +1,81 @@
+/*! Patterns - Utilities - Provider Login Button */
+////
+/// @group Utilities-Provider_Login_Button
+/// Patterns - Utilities - Provider Login Button
+
+.btn {
+	&.btn-provider-login {
+		background: var(--color-neutral-0);
+		border-color: var(--color-neutral-6);
+		border-radius: 0;
+		&-logo {
+			&-only {
+				padding: 10px;
+				.btn-provider-login-text {
+					display: none;
+					-servicestudio-display: none !important;
+				}
+			}
+			svg {
+				height: 100%;
+				width: 100%;
+			}
+		}
+		&.btn- {
+			&small {
+				padding: 0 16px;
+				&.btn-provider-login-logo-only {
+					padding: 5px;
+				}
+			}
+			&large {
+				padding: 10px 15px;
+			}
+		}
+		&.soft {
+			border-radius: var(--border-radius-soft);
+		}
+		&.rounded {
+			border-radius: var(--border-radius-rounded);
+		}
+		.btn-provider-login {
+			&-logo {
+				height: 20px;
+				width: 20px;
+				border-radius: 50%;
```

Comment:
Shouldn't we organize the CSS alphabetically?

Reply:
Absolutely! Thank you!

Reply:
Fixed. Please validate.